### PR TITLE
Refactor JavaScript Code to ES6 Modules

### PR DESCRIPTION
**Pull request by AyoubM490**
In this milestone, I:
- refactored the old Awesome Books project from standard ES6 classes to ES6 Modules
- Used luxon for the date display

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
+import { Book } from '../modules/Book.js';
+import { DateTime } from '../modules/luxon.js';
+
+const addBtn = document.getElementById('addBtn');
+const title = document.getElementById('title');
+const author = document.getElementById('author');
+
+function addBook(e) {
```
Comment (elmar8287) [inline js/index.js]: - [x] Please, consider to use arrow function syntax 👍

File: modules/Book.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  addNewBook() {
+    Book.books.push({ title: this.title, author: this.author });
+    this.store.setItem('books', JSON.stringify(Book.books));
+  }
+
+  static removeBook(title) {
+    Book.books = Book.books.filter((book) => book.title !== title);
+    this.store.setItem('books', JSON.stringify(Book.books));
+  }
+}
```
Comment (elmar8287) [inline modules/Book.js]: - [x] The book removing method is not working. Please, try to fix it:

**Review by elmar8287 — CHANGES_REQUESTED** [review]

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi @AyoubMs, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [x]  Check the comments under the review.
- [x] Please, consider fixing the demo live link - it is broken.

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/index.js (hunk 1 above)
Comment (AyoubM490) [reply js/index.js]: Done

File: modules/Book.js (hunk 2 above)
Comment (AyoubM490) [reply modules/Book.js]: Done

File: js/index.js (hunk 1 above)
Comment (AyoubM490) [reply js/index.js]: Done

File: modules/Book.js (hunk 2 above)
Comment (AyoubM490) [reply modules/Book.js]: Done

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @AyoubMs, 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**